mobile/android/android-emulator-services.ts

```diff
@@ -365,7 +365,11 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 		var platform = target.replace('android-', '');
 		var platformNumber = +platform;
 		if (isNaN(platformNumber)) {
-			if (platform === "L") {
+			// this might be a google image
+			platform = target.split(":")[2];
+			if (platform) {
+				platformNumber = +platform;
```

Comment:
What will happen in case this is some strange image and after splitting by ":" we get string or undefined for platform. +ing it will set platformNumber ot NaN and the code will not work